azure-iot-device/tests/provisioning/pipeline/test_pipeline_stages_provisioning_mqtt.py

```diff
@@ -79,6 +79,7 @@ def apply_fake_pipeline_thread(fake_pipeline_thread):
     handled_ops=ops_handled_by_this_stage,
     all_events=all_common_events + all_provisioning_events,
     handled_events=events_handled_by_this_stage,
+    extra_initializer_defaults={"action_to_topic": dict},
```

Comment:
Same as the comment in `test_pipeline_stages_iothub_mqtt.py`

Reply:
same as the response in test_pipeline_stages_iothub_mqtt.py

---
In reply to: [312593655](https://github.com/Azure/azure-iot-sdk-python-preview/pull/160#discussion_r312593655) [](ancestors = 312593655)